enclave/core/sgx/asmdefs.h

```diff
@@ -38,9 +38,9 @@
 #define td_callsites (td_oret_arg + 8)
 #define td_simulate (td_callsites + 8)
 
-#define oe_exit __morestack
+#define oe_exit_enclave __morestack
 #ifndef __ASSEMBLER__
-void oe_exit(uint64_t arg1, uint64_t arg2);
```

Comment:
Note to reviewers: oe_exit() was renamed to avoid a conflict with the one in stdlib.h.

Reply:
I'd suggest adding a /* comment */ in the code explaining the difference

Reply:
Done. Thanks Dave.